### PR TITLE
json: Clean up docs

### PR DESCRIPTION
**Pull request by ibgreen**
extracted from the WKT PR #575

### CODE REVIEW

**Comment by ibgreen** [comment]
@xintongxia Time to publish new website for 2.0?

**Comment by xintongxia** [comment]
> @xintongxia Time to publish new website for 2.0?

I have tried, however there are some issues, not sure it is related to gastby, wonder if you have time to help or maybe we can sync up about how to debug in gastby.

File: modules/json/docs/README.md [hunk 1]
```diff
@@ -1,75 +1,37 @@
...
-
-const data = await load(url, JSONLoader, {json: options});
+```bash
+npm install @loaders.gl/core @loaders.gl/json
 ```
 
-The JSONLoader supports streaming JSON parsing, in which case it will yield "batches" of rows of the first array it encounters in the JSON. To e.g. parse a stream of GeoJSON:
-
-```js
-import {JSONLoader} from '@loaders.gl/json';
```
Comment (xintongxia) [inline modules/json/docs/README.md]: Is there other places mentioning how to use this loader?

Comment (ibgreen) [reply modules/json/docs/README.md]: Yes the structure is that the module README just mentions which classes the module provides, and the `api-reference` folder contains a reference for each class/loader.

I am also doing background work on improved user guides for `parseInBatches` and table category.

**Pull request merged**